docs/references/django_model_types.md

```diff
@@ -0,0 +1,49 @@
+# Django Model Types
+
+Django Models can be converted in Strawberry Types with the `@strawberry_django.type` decorator
+
+```python
+import strawberry
+
+@strawberry.django.type(models.Fruit)
+class Fruit:
+    ...
+```
+
+## Fields
+
+By default, no fields are implemented on the new type. For details on adding fields,
+see the [Fields](fields.md) documentation.
+
+## Queryset setup
+
+By default, a Strawberry Django type will pull data from the default manager for it's Django Model.
+You can change this behavior by implementing a `get_queryset` method on the type.
+
+```python
+@strawberry.django.type(models.Fruit)
+class Berry:
+
+    def get_queryset(self, queryset, info):
+        return queryset.filter(name__contains="berry")
+```
+
+The `get_queryset` method is given a Queryset to filter and
+a Strawberry `Info` object, containing details about the request.
+
+You can use the `info` parameter to limit access to items.
```

Comment:
Maybe "You can use that `info` parameter to, for example, limit access to results based on the current user in the request:"